launcher/src/backend/ValidatorAccountManager.js

```diff
@@ -25,17 +28,54 @@ export class ValidatorAccountManager {
       return readFileSync(file.path, { encoding: "utf8" });
     });
     const chunkSize = 100;
-    let batches = [];
     for (let i = 0; i < content.length; i += chunkSize) {
       const contentChunk = content.slice(i, i + chunkSize);
       const passwords = Array(contentChunk.length).fill(password);
-      batches.push({
+      this.batches.push({
         keystores: contentChunk,
         passwords: passwords,
         ...(slashing_protection_content && { slashing_protection: slashing_protection_content }),
       });
     }
-    return batches;
+    return this.batches;
+  }
+
+  async checkActiveValidators(files, password, serviceID, slashingDB) {
+    this.batches = [];
+    this.batches = this.createBatch(files, password, slashingDB);
+    let services = await this.serviceManager.readServiceConfigurations();
+    let client = services.find((service) => service.id === serviceID);
+
+    let pubkeys = this.batches.map((b) => b.keystores.map((c) => JSON.parse(c).pubkey)).flat();
+    let isActiveRunning = [];
+
+    if (pubkeys.length < 11) {
+      let networkURLs = {
+        mainnet: "https://mainnet.beaconcha.in/api/v1/validator/",
+        goerli: "https://goerli.beaconcha.in/api/v1/validator/",
+        gnosis: "https://beacon.gnosischain.com/api/v1/validator/",
+      };
+      try {
+        for (const pubkey of pubkeys) {
+          let latestEpochsResponse = await axios.get(networkURLs[client.network] + pubkey + "/attestations");
+          if (
+            latestEpochsResponse.status === 200 &&
+            latestEpochsResponse.data.data.length > 0 &&
+            latestEpochsResponse.data.status !== /ERROR:*/
+          ) {
+            for (let i = 0; i < 2; i++) {
+              if (latestEpochsResponse.data.data[i].status === 0 && isActiveRunning.indexOf(pubkey) === -1) {
```

Comment:
guys you have to test your stuff properly 😅
`status` `1` means attested...
I was able to import an active validator on another machine without a problem.

Reply:
true, that was for testing purpose, :)

Reply:
tnx